kickstart/simulator-fixes.yaml

```diff
@@ -0,0 +1,20 @@
+summary: Work through open issues against the simulator
```

Comment:
This doesn't feel like it needs to be a recurring task. If it does, we should have a similar one for the kit.

Reply:
This is a process miss we've repeatedly hit. Conversely, the kit has a separate team that has an explicit responsibility to maintain the kit.